site/documentation/content/deployment/helm-based-deployment.md

```diff
@@ -108,16 +108,11 @@ helm install --dependency-update -n hono --set honoImagesTag=$HONO_VERSION eclip
 
 ### Using Jaeger Tracing
 
-Hono's components are instrumented using OpenTracing to allow tracking of the distributed processing of messages
-flowing through the system. The Hono chart can be configured to report tracing information to the
-[Jaeger tracing system](https://www.jaegertracing.io/). The *Spans* reported by the components can then be viewed in a
-web browser.
-
-In order for Hono's components to use the Jaeger client for reporting tracing information, the container images need
-to be built with the `jaeger` Maven profile. Please refer to
-[Monitoring & Tracing]({{< relref "/admin-guide/monitoring-tracing-config#configuring-usage-of-jaeger-tracing-included-in-docker-images" >}})
-for details. The newly built images also need to be made available to the target Kubernetes cluster as described in the
-two previous sections.
+Hono's components are instrumented using OpenTelemetry to allow tracking of the distributed processing of messages
+flowing through the system. The tracing information of a Hono component is exported in the OpenTelemetry OTLP format
+to an OpenTelemetry collector agent sidecar container. The Hono chart can be configured so that the collectors report 
+tracing information to the [Jaeger tracing system](https://www.jaegertracing.io/). The *Spans* reported by the
+components can then be viewed in a web browser.
```

Comment:
I guess we should move most of this section to the Helm chart's documentation as it  directly relies on the way that the chart is implemented, right?

Reply:
Right. This chapter is already contained in the Helm chart documentation, so I've removed it here.